doc/source/cluster/kubernetes/doc_sanitize.cfg

```diff
@@ -0,0 +1,27 @@
+[kuberay-operator-pod-name]
+regex: kuberay-operator-[a-z0-9]{10}-[a-z0-9]{5}
+replace: KUBERAY-OPERATOR-POD-NAME
+
+[raycluster-head-pod-name]
+regex: raycluster-kuberay-workergroup-worker-[a-z0-9]{5}
```

Comment:
For KubeRay v1.4.0, the naming changes a bit. This may require some updates in the future.

Reply:
Yes we need to go back to change this doc after 1.4.0 release. But for now let's keep the naming here.